crates/net/src/sim2h_worker.rs

```diff
@@ -389,15 +357,12 @@ impl NetWorker for Sim2hWorker {
 
     /// Check for messages from our NetworkEngine
     fn tick(&mut self) -> NetResult<bool> {
-        if Instant::now().duration_since(self.time_of_last_sent) > Duration::from_secs(PING_DURATION_SECS) {
-            self.send_ping();
-        }
-        if let Err(transport_error) = detach_run!(&mut self.transport, |t| t.process(self)) {
-            error!("Transport error: {:?}", transport_error);
-            // This most likely means we have connection issues.
-            // Send ping to reestablish a potentially lost connection.
+        if Instant::now().duration_since(self.time_of_last_sent)
```

Comment:
Without the websocket actor, this ping doesn't make sense any more and should either just go away, or better: we actually check if the connection is still there and try to reconnect immediately if we lost it..